common/src/main/java/com/microsoft/hydralab/performance/PerformanceTestManagementService.java

```diff
@@ -34,19 +28,24 @@ public class PerformanceTestManagementService implements IPerformanceInspectionS
             INSPECTOR_ANDROID_BATTERY_INFO, PARSER_ANDROID_BATTERY_INFO,
             INSPECTOR_WIN_MEMORY, PARSER_WIN_MEMORY,
             INSPECTOR_WIN_BATTERY, PARSER_WIN_BATTERY,
-            INSPECTOR_ANDROID_MEMORY_INFO, PARSER_ANDROID_MEMORY_INFO
```

Comment:
Why not remove INSPECTOR_ANDROID_MEMORY_INFO?  It caused NullPointerException for Android memory test!